.changeset/dry-birds-peel.md

```diff
@@ -0,0 +1,7 @@
+---
+"modern-walk-react": patch
+---
+
+- Disabled submit button and displays loading message while loading in sign in and sign up page
+- Centered logo in navigation bar
+- Limited the amount of products displayed in flash sale to four
```

Comment:
@siroshan Let's merge auth PR before this—since this changeset is already in it, we can remove it from here!